VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -137,8 +135,8 @@ public void updateDrawState(TextPaint ds) {
                         context.getResources(), finalStringRes, valuesMap));
                 // If time is not available, then reduce the whitespaces
                 // surrounding it's placeholder in the template to one.
-                if (TextUtils.isEmpty(formattedText) || TextUtils.isEmpty(authorLabel)) {
-                    formattedText = removeDuplicateWhitespace(formattedText);
+                if (TextUtils.isEmpty(formattedTime)) {
+                    formattedText = removeDuplicateWhitespaces(formattedText);
```

Comment:
hmmm .. It now comes down to deciding between the following 3 solutions:
1. Have a total of 18 string templates for every combination.
2. Localise the `whitespace` string i.e. `<string name="space"> </string>` and use it to replace its duplicates.
3. Go by iOS way.

Which one do you suggest ?